docs/learn/learn-comparison-rollups.md

```diff
@@ -0,0 +1,159 @@
+
+## Polkadot - Native Shared Security
+
+Whereas rollups are considered solutions for L2 protocols,
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} include this functionality natively
+through its [Parachains Protocol](./learn-parachains-protocol.md). The Parachains Protocol, which is
+how {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} handles network's **sharding** is
+meant to accomplish the combined goals of providing security, scalability, and availability.
+
+It enables parachains to verify their collective state and communicate with one another. Parachains
+have similarities to aspects of optimistic and ZK rollups, which are reflected in how
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} handles the validity and availability
+of the parachain state. [Collators](./learn-collator.md), a key part of
+{{ polkadot: Polkadot's :polkadot }}{{ kusama: Kusama's :kusama }} architecture, is in principle
+similar to a sequencer, as collators pass data with a proof-of-validity (PoV) function for liveness
+and communication with the relay chain.
+
+Each shard, or parachain, is equipped with a unique state transition function (STF). This function
+ensures that communication to the relay chain remains valid. Each STF, called runtime, is written in
+[Wasm](https://wiki.polkadot.network/docs/learn-wasm). Any state transition function is valid if it
+compiles to Wasm and abides by the Parachains Protocol.
+
+Each STF runs a validity proof. The proof ([the Approval Protocol](./learn-availability.md)) is
+interactive, unlike ZK rollups, which are non-interactive. Additionally, unlike ZK rollups, there
+are no difficulties in creating parachains with Turing-complete logic. Each parachain is also a
+full-fledged state machine (usually in the form of a blockchain). Similarly to optimistic rollups,
+the Parachain Protocol also has cases where disputes and resolutions of potentially harmful para
+blocks (blocks representing the parachain) can take place, in which case validators are slashed if a
+bad parablock is found.
+
+**Benefits:**
+
+- Protocol level sharding, shared security, and interoperability.
+- Each shard has a low barrier of entry in terms of development, as anything that compiles to Wasm
+  is a valid target.
+- Fast Finality (usually under a minute on Polkadot).
+- Data availability is built-in through validators and mechanisms like
+  [erasure coding](./learn-availability#erasure-codes).
+- No L2 implies less of a risk of incurring centralization issues for sequencers or other L2
+  operators.
+
+**Drawbacks:**
+
+- Wasm could be a performance bottleneck, as it is slower than making native calls.
+- A considerable amount of data is required in the PoV (proof of validity) function for proving the
+  parachain state.
+- Other limitations within the Parachains Protocol to keeping parachains compatible with the entire
+  orchestration of the relay and parachains.
```

Comment:
This point is too vague and can mislead the reader into assuming there are inefficiencies in the design. Unless there is a concrete drawback, avoid general statements like these.
```suggestion
```